### DISCUSSION TITLE
window_placement first_window

### DISCUSSION THREAD

**Discussion by Bellavene** [asmvik/yabai#2508] [Ideas]
Could we have as a Christmas gift more window placement options? Especially first and last window

**Comment by asmvik** [comment]
I can see how something like `config window_insertion_point focused|first|last` would make sense.

**Reply by Bellavene** [reply]
This will make things so much simpler, I've tried every way to script it

**Reply by asmvik** [reply]
Implemented on master, track #2510

**Reply by Bellavene** [reply]
Thank You!

**Reply by Bellavene** [reply]
For some reason, I got no `yabai -m config window_insertion_point` option, but new `--space split_type` exist.
Tried every way to install Yabai
`brew upgrade yabai --force --fetch-HEAD`, ` brew install yabai --fetch-HEAD` and locally.

**Reply by asmvik** [reply]
Not sure what is happening there. It works fine on my end, but I'm not using brew.

**Reply by Bellavene** [reply]
With `curl -L https://raw.githubusercontent.com/koekeishiya/yabai/master/scripts/install.sh | sh /dev/stdin`
I got the same
```
unknown command 'window_insertion_point' for domain 'config'
unknown command 'first' for domain 'config'
```

**Reply by Bellavene** [reply]
Installed it locally again, now it works. For some reason, I had to recreate the keychain certificate. Thank you
